src/Surfnet/AzureMfa/Domain/User.php

```diff
@@ -0,0 +1,77 @@
+<?php declare(strict_types = 1);
+
+/**
+ * Copyright 2019 SURFnet B.V.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+namespace Surfnet\AzureMfa\Domain;
+
+class User
```

Comment:
Why is this ValueObject not covered by a unit test? Granted, the complexity is very low, but a test still might prove usefull in the future.

Reply:
Because it's a simple VO and there is no business logic whatsoever, if the DI-ed VO's are valid this object is valid.